policy-design-lab/pdl-frontend
# Release 1.0.1: Percentage map, list switch feature and NaN error fix

This PR covers issues #285, #286, and #290. It includes the following features: adding percentage numbers to the map tips, switching the practice list when users switch between present and predicted data, and fixing the NaN display for some percentage calculations.

## How to Test

1. Navigate to the IRA page and hover over the map. You should see 'PCT. Nationwide' displayed, which should match the Percentage table under the map:

<img width="983" alt="Screenshot 2024-07-18 at 5 20 12 PM" src="https://github.com/user-attachments/assets/131c625a-00a4-4476-88c6-de412c263902">


2. Select a few practices, including some that are not available for predicted years (such as Forest Management Practice Design). You should see a total percentage calculation for all practices appear in the map tips, also matching the Percentage table under the map:

<img width="718" alt="Screenshot 2024-07-18 at 5 23 06 PM" src="https://github.com/user-attachments/assets/b6c4c0a9-86ea-4767-86d4-8ae68ef5621d">


3. Turn on the '2024-2031 Prediction' switch. You should see any practices that are not available in predicted years (such as Forest Management Practice Design) disappear from the map. The map and tables will update to reflect these changes.

4. Repeat steps 1 and 2 for the predicted data. No NaN value should appear in any tables or maps.

5. Turn the '2024-2031 Prediction' switch off and ONLY select practices that are not available in predicted years. Then turn the switch on, and you should see 'Total' being automatically selected in the practice list because all other practices have been removed.
